vpr/src/place/place.cpp

```diff
@@ -2446,7 +2443,7 @@ static int check_block_placement_consistency() {
                 }
```

Comment:
One code related item I thought of, and don't see any changes related to. Should there be updates to the placement consistency checks given that tiles can now be of heterogeneous types?

It looks like `check_block_placement_consistency()` is unchanged with respect to this. I think the above block type check will need to be generalized to verify that the sub-tile location (not just the outer tile) is type consistent.

I should add, that if we haven't hit this issue in the add `strong_sub_tile` regression test it likely means we need more test cases which would exercise this case.